packages/angular_devkit/build_optimizer/src/transforms/scrub-file.ts

```diff
@@ -304,7 +303,35 @@ function isAssignmentExpressionTo(exprStmt: ts.ExpressionStatement, name: string
 }
 
 function isIvyPrivateCallExpression(exprStmt: ts.ExpressionStatement) {
```

Comment:
nit: The IIFE structure could technically be different depending on any intermediate processing steps.  The CLI's fixed configuration will prevent this but other usage may cause changes.  An unbounded number of parenthesized expressions are also technically possible.

Examples: `!function(){}()` or `(function(){}())` or `((function(){})())`